src/features/virtualized-table/DraggableVirtualizedTable.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import * as React from 'react';
+import noop from 'lodash/noop';
+import classNames from 'classnames';
+import { Column } from 'react-virtualized/dist/es/Table/index';
+import { DragDropContext, Droppable } from 'react-beautiful-dnd';
+
+import { black } from '../../styles/variables';
```

Comment:
Most things shouldn't be pure black; here we may actually want #222 (`$bdl-gray`). Can you confirm with design on this?